assets/js/script.js

```diff
@@ -0,0 +1,429 @@
+  win.style.display = 'none'
+  lose.style.display = 'none'
+
+  blueGhost.direction = 'left'
+  orangeGhost.direction = 'right'
+  pinkGhost.direction = 'up'
+  redGhost.direction = 'down'
+
+  pacman.style.top = '440px'
+  pacman.style.left = '280px'
+  pacman.style.height = '40px'
+  pacman.style.width = '40px'
+
+  blueGhost.style.top = '40px'
+  blueGhost.style.left = '40px'
+  blueGhost.style.height = '40px'
+  blueGhost.style.width = '40px'
+
+  orangeGhost.style.top = '40px'
+  orangeGhost.style.left = '520px'
+  orangeGhost.style.height = '40px'
+  orangeGhost.style.width = '40px'
+
+  pinkGhost.style.top = '440px'
+  pinkGhost.style.left = '40px'
+  pinkGhost.style.height = '40px'
+  pinkGhost.style.width = '40px'
+
+  redGhost.style.top = '440px'
+  redGhost.style.left = '520px'
+  redGhost.style.height = '40px'
+  redGhost.style.width = '40px'
+
+  // function to create walls for gameboard
+  function createWall (left, top, width, height) {
+    var wall = document.createElement('div')
+    wall.className = 'wall'
+    wall.style.left = left + 'px'
+    wall.style.top = top + 'px'
+    wall.style.width = width + 'px'
+    wall.style.height = height + 'px'
+    gameboard.appendChild(wall)
+    walls.push(wall)
+  }
+
+  // gameboard borders
+  createWall(0, 0, 600, 40)
+  createWall(0, 0, 40, 280)
+  createWall(0, 280, 40, 200)
+  createWall(0, 480, 600, 40)
+  createWall(560, 0, 40, 280)
+  createWall(560, 280, 40, 240)
+  // gameboard tiles
+  // W
+  createWall(80, 80, 40, 120)
+  createWall(160, 120, 40, 80)
+  createWall(240, 80, 40, 120)
+  createWall(80, 200, 200, 40)
+  // D
+  createWall(320, 80, 40, 160)
+  createWall(360, 80, 40, 40)
+  createWall(360, 200, 40, 40)
+  createWall(400, 120, 40, 80)
+  // I
+  createWall(480, 80, 40, 160)
+  // 11
+  createWall(240, 280, 40, 160)
+  createWall(320, 280, 40, 160)
+  // others
+  createWall(40, 280, 160, 40)
+  createWall(400, 280, 120, 40)
+  createWall(80, 360, 40, 80)
+  createWall(160, 360, 40, 80)
+  createWall(400, 360, 40, 80)
+  createWall(480, 360, 40, 80)
+
+  // function to create dots for gameboard
+  function createDot (left, top) {
+    var dot = document.createElement('div')
+    dot.className = 'dot'
+    dot.style.left = left + 'px'
+    dot.style.top = top + 'px'
+    dot.style.width = 8 + 'px'
+    dot.style.height = 8 + 'px'
+    dot.style.borderRadius = 5 + 'px'
+    gameboard.appendChild(dot)
+    dots.push(dot)
+  }
+
+  // create dots
+  for (var i = 40; i < 540; i += 40) {
+    for (var j = 40; j < 480; j += 40) {
+      createDot(i + 16, j + 16)
+    }
+  }
+
+  // remove the dot at pacman's starting point and two inside 'D'
+  gameboard.removeChild(dots[76])
+  gameboard.removeChild(dots[90])
+  gameboard.removeChild(dots[91])
+  dots.splice(76, 1)
+
+  // key down event listener
+  document.addEventListener('keydown', function (event) {
+    // up key
+    if (event.keyCode === 38) {
+      upKeyDown = true
+      downKeyDown = false
```

Comment:
"downKeyDown = false" and following are already declared in global function so I think these are not needed. Unless you want to redefine them. So you won't have to repeat the 'false' declarations.